src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -38,47 +38,78 @@
  */
 class TextDescriptor extends Descriptor
 {
+    private const VERB_COLORS = [
+        'ANY' => 'default',
+        'GET' => 'blue',
+        'HEAD' => 'magenta',
+        'OPTIONS' => 'blue',
+        'POST' => 'green',
+        'PUT' => 'yellow',
+        'PATCH' => 'yellow',
+        'DELETE' => 'red',
+    ];
+
     public function __construct(
         private ?FileLinkFormatter $fileLinkFormatter = null,
     ) {
     }
 
     protected function describeRouteCollection(RouteCollection $routes, array $options = []): void
     {
-        $showControllers = isset($options['show_controllers']) && $options['show_controllers'];
-
-        $tableHeaders = ['Name', 'Method', 'Scheme', 'Host', 'Path'];
-        if ($showControllers) {
-            $tableHeaders[] = 'Controller';
-        }
-
-        if ($showAliases = $options['show_aliases'] ?? false) {
-            $tableHeaders[] = 'Aliases';
-        }
+        $showAliases = $options['show_aliases'] ?? false;
 
         $tableRows = [];
+        $shouldShowSchema = false;
```

Comment:
it's not schema but scheme
this change should be done everywhere in the PR, including file names